osmaxx-py/config/settings/common.py

```diff
@@ -198,6 +202,7 @@
 
 MEDIA_ROOT = env.str('DJANGO_MEDIA_ROOT', default=str(ROOT_DIR('..', 'media')))
 PRIVATE_MEDIA_ROOT = env.str('DJANGO_PRIVATE_MEDIA_ROOT', default=str(ROOT_DIR.path('..', 'private_media')))
+RESULT_MEDIA_ROOT = '/results'
```

Comment:
Path `/results` is only acceptable inside a Docker container. Use `env` to parametrize this setting.